src/openvic-simulation/modifier/StaticModifierCache.cpp

```diff
@@ -1,115 +1,166 @@
 #include "StaticModifierCache.hpp"
-
+#include <cstddef>
+#include <string_view>
 #include "openvic-simulation/modifier/ModifierManager.hpp"
+#include "openvic-simulation/dataloader/NodeTools.hpp"
+#include "modifier/Modifier.hpp"
```

Comment:
```suggestion

#include <cstddef>
#include <string_view>

#include "openvic-simulation/dataloader/NodeTools.hpp"
#include "openvic-simulation/modifier/Modifier.hpp"
#include "openvic-simulation/modifier/ModifierManager.hpp"
```
Include formatting: newlines between groups, alphabetical order, full paths